src/main/scala/Utilities.scala

```diff
@@ -0,0 +1,40 @@
+import org.apache.log4j.Level
+import java.util.regex.Pattern
+import java.util.regex.Matcher
+
+object Utilities {
+  /** Makes sure only ERROR messages get logged to avoid log spam. */
+  def setupLogging() = {
+    import org.apache.log4j.{Level, Logger}
+    val rootLogger = Logger.getRootLogger()
+    rootLogger.setLevel(Level.ERROR)
+  }
+
+  /** Configures Twitter service credentials using twiter.txt in the main workspace directory */
+  def setupTwitter() = {
+    import scala.io.Source
+
+    for (line <- Source.fromFile("resources/creds.txt").getLines) {
+      val fields = line.split("=")
+      if (fields.length == 2) {
+        System.setProperty("twitter4j.oauth." + fields(0), fields(1))
+      }
+    }
+  }
+
+  /** Retrieves a regex Pattern for parsing Apache access logs. */
+  def apacheLogPattern():Pattern = {
```

Comment:
is there a ready-to-use lib for this purpose? looks like a common problem.

Reply:
I tried to google this, but found only similar solutions.